web_src/js/features/repo-actions.js

```diff
@@ -0,0 +1,8 @@
+export function initRepoActionList() {
+  const disableWorkflowBtn = document.getElementById('disable-workflow-btn');
+  if (disableWorkflowBtn) {
+    disableWorkflowBtn.addEventListener('click', () => {
+      document.getElementById('disable-workflow-form').submit();
+    });
+  }
```

Comment:
No need, please use `link-action`

Reply:
Got an example?

Reply:
The examples on the devtest page:

http://localhost:3000/devtest/fetch-action

![image](https://github.com/go-gitea/gitea/assets/2114189/4268b7bb-e746-4bcc-a1ad-6f66a8e089d5)

Reply:
Was fixed in https://github.com/go-gitea/gitea/pull/26413/commits/70d758b2ae01f12812fc772e2e15cc60ddea2caa.